Task-based edit: Security Settings (#6079)
braze-inc/braze-docs: _docs/_user_guide/administrative/app_settings/company_settings/security_settings.md

To download this report, do the following:

1. Go to your user profile and select **Company Settings**.
2. Select the **Security Settings** tab and go to the **Security Event Download** section.
2. Select **Download report**. 

This report only contains the most recent 10,000 security events for your account. If you need specific event data, contact technical support.

## Viewing personally identifiable information (PII)

The **View PII** permission is only accessible to a few select Braze users. For the existing team permission capabilities, see [Setting user permissions]({{site.baseurl}}/user_guide/administrative/manage_your_braze_users/user_permissions/#available-limited-and-team-role-permissions).

By default, all admins have their **View PII** permission enabled in [user permissions]({{site.baseurl}}/user_guide/administrative/manage_your_braze_users/user_permissions/#available-limited-and-team-role-permissions). This means they can see the following standard and custom attributes throughout the dashboard. When this permission is disabled for users, those users will not be able to see this information.

### Defining PII

> Braze allows you to define which fields are designated as PII in your dashboard. To do this, navigate to **Company Settings > Security Settings**.

Reviewer: ```suggestion
You can define which fields are designated as PII in the dashboard. To do this, navigate to **Company Settings** > **Security Settings**.
```
